tests/Client/Client.Tests.fs

```diff
@@ -16,7 +17,7 @@ let all =
 #if FABLE_COMPILER // This preprocessor directive makes editor happy
             Shared.Tests.shared
 #endif
-            OfficeAddin.Tests.Main
+            Successful.Main
```

Comment:
If you want to reuse the "Successful" syntax for other tests you MUST NOT open the path to these tests, as you would shadow your "Main" otherwise

Reply:
fixed